code/game/gamemodes/dynamic/dynamic_readme.dm

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
The readme hasn't been updated to include the changes done in this PR, that must be done later.
Only change done to it was remove the faction mentions.